src/components/chat-view-container/styles.scss

```diff
@@ -17,11 +18,14 @@
   &__message-input-container {
     margin-bottom: 16px;
     margin-right: 16px;
-    margin-top: 16px;
     border-radius: 16px;
     overflow: hidden;
     flex-shrink: 0;
-    background-color: theme.$color-primary-1;
+
+    @include flat-thick;
+
+    // todo: remove this after extracting out the message input from chat-view
+    backdrop-filter: none;
```

Comment:
added this for now, so that the emoji picker can appear. Shall remove this after extracting out `<MessageInput>` from `<ChatView>`, and move it higher up the DOM container